elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -445,30 +446,28 @@ def create_pipeline_file(self, pipeline, pipeline_export_format, pipeline_export
 
         return pipeline_export_path
 
-    def _create_unique_node_names(self, operation_list, parent_unique_names=None):
-        unique_names = parent_unique_names or {}
+    def _create_unique_node_names(self, operation_list: List[Operation]) -> List[Operation]:
+        unique_names = {}
         for operation in operation_list:
             # if a duplicate name is found
             if operation.name in unique_names:
```

Comment:
With the last simplification, this `if` could be removed.  This would save a "contains" check on each iteration, but probably not a big deal.  The comment could probably be updated to something like `# Ensure operation name is unique` which would then apply to the block.

Reply:
done!